webview-codicons-sample/src/extension.ts

```diff
@@ -43,15 +57,40 @@ class CatCodiconsPanel {
 					Use a content security policy to only allow loading images from https or from our extension directory,
 					and only allow scripts that have a specific nonce.
 				-->
-				<meta http-equiv="Content-Security-Policy" content="default-src 'none'; font-src ${codiconsFontUri}; style-src ${webview.cspSource} ${codiconsUri};">
+				<meta http-equiv="Content-Security-Policy" content="default-src 'none'; font-src ${codiconsFontUri}; style-src ${webview.cspSource} ${codiconsUri}; script-src 'nonce-${nonce}'">
 
 				<meta name="viewport" content="width=device-width, initial-scale=1.0">
-				<title>Cat Coding</title>
+				<title>Webview Coding</title>
 
 				<link href="${styleUri}" rel="stylesheet" />
 				<link href="${codiconsUri}" rel="stylesheet" />
 			</head>
 			<body>
+
+				<h1>styled codicons</h1>
+
+				<div id="icons">
+					<div class="styledIcon" title="This icon is styled with green foreground"><i class="codicon codicon-archive"></i></div>
+					<div class="icon" title="This icon is styled with continuous rotation"><i class="codicon codicon-loading rotate"></i></div>
+				</div>
+
+				<h1>codicons mouse events</h1>
```

Comment:
To keep this example simple, I don't think we should include mouse events or buttons.

We want this to just be a basic: here's how you can use codicons in a webview

Reply:
Javascript files removed for the sake of simplicity. The styling and interaction sample moved to the bottom as it indeed is optional. Do you still it is making the sample less useful?